schema_utils.py

```diff
@@ -387,3 +411,89 @@ def is_valid_email(obj):
             bool. Whether the given object is a valid email.
         """
         return bool(re.search(r'^[\w\.\+\-]+\@[\w]+\.[a-z]{2,3}$', obj))
+
+    @staticmethod
+    def is_valid_expression(obj, algebraic=True):
+        """Checks if the given  obj (a string) represents a valid algebraic or
+        numeric expression.
+
+        Args:
+            obj: str. The given expression.
+            algebraic: bool. True if the given expression is algebraic
+            else numeric.
+
+        Returns:
+            bool. Whether the given object is a valid expression.
+        """
+        valid_expression = True
```

Comment:
expression_is_valid

But I think you can do away with this, and just "return False" early at various places? Easier to follow, that way.

Reply:
Done.